_security/multi-tenancy/multi-tenancy-config.md

```diff
@@ -21,6 +21,7 @@ config:
       index: '.kibana'
     do_not_fail_on_forbidden: false
 ```
+The configuration mentioned is for OpenSearch Dashboards, which inherits much of Kibana's functionality but is now developed and maintained as part of the OpenSearch project.
```

Comment:
We can probably move this sentence to line 11.

Something like, "Multi-tenancy is enabled in OpenSearch Dashboards by default. If you need to disable or change settings related to multi-tenancy, see the `kibana` settings in `config/opensearch-security/config.yml`, as shown in the following example:"

Reply:
Great thank you for the suggestion @Naarcha-AWS. I have made that edit in line with your suggestion.